README.md

```diff
@@ -31,7 +31,7 @@ sources into WebAssembly.
 
 ![Build Process](./wasm-readme-images/build-process.png)
 
-WebAssembly frontend for Vim is implemented as a new GUI frontend.  C sources are
+WebAssembly frontend for Vim is implemented as a new GUI frontend. C sources are
```

Comment:
Two spaces after period are rule of [original README](https://github.com/vim/vim/blob/master/README.md). I don't want to mix multiple rules in one README file (maximum 80 characters per line is also a rule of original README). So please keep two spaces.